packages/devtools_app/pubspec.yaml

```diff
@@ -129,3 +129,5 @@ dependency_overrides:
     path: ../devtools_shared #OVERRIDE_FOR_DEVELOPMENT
   devtools:  #OVERRIDE_FOR_DEVELOPMENT
     path: ../devtools  #OVERRIDE_FOR_DEVELOPMENT
+  widget_icons: #OVERRIDE_FOR_DEVELOPMENT
+    path: ../../third_party/packages/widget_icons  #OVERRIDE_FOR_DEVELOPMENT
```

Comment:
we should remove this. Once we publish 0.0.1 we can depend on it normally in the main dependency sections above. For now add a widget_icons dep in the mail dependencies section and depend on it from path there.

Reply:
Fixed